dev-docs/bidders/loyal.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: bidder
+title: Loyal
+description: Prebid Loyal Bidder Adapter
+biddercode: loyal
+usp_supported: true
+tcfeu_supported: false
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
+multiformat_supported: will-not-bid
+pbjs: true
+pbs: true
+pbs_app_supported: true
+safeframes_ok: true
+deals_supported: false
+ortb_blocking_supported: false
+fpd_supported: false
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description  | Example                         | Type       |
+|---------------|----------|--------------|---------------------------------|------------|
+| `placementId` | optional | Placement Id | `'0'`                           | `'string'` |
+| `endpointId`  | optional | Endpoint Id  | `'0'`                           | `'string'` |
```

Comment:
should include `bidfloor`

Reply:
our adapter does not have the bidfloor parameter. Why do we need to add it?

Reply:
Looks you have it included in your bid adapter submission PR. If this is not intended, can it be removed?

![image](https://github.com/prebid/prebid.github.io/assets/34142758/7260fd85-d645-4f36-a4df-f730855ccf3d)
https://github.com/prebid/Prebid.js/pull/11253/files

Reply:
removed